### PR TITLE
Bump version for shadow jar

### PR DESCRIPTION
**Pull request by nwbirnie**
Change-Id: I93c57f4d1976e568a92e89feafa5248f982cd2b3

### CODE REVIEW

File: google-ads/build.gradle [hunk 1]
```diff
@@ -369,7 +369,7 @@ publishing {
 
       groupId = 'com.google.api-ads'
       artifactId = 'google-ads-shadowjar'
-      version = '0.0.1-alpha'
+      version = '1.0.0'
```
Comment (jradcliff) [inline google-ads/build.gradle:372]: If this is ready to move out of the alpha phase, should the version match the overall library version? I'm concerned another versioning scheme could be confusing on top of the library and API versions.

Comment (nwbirnie) [reply google-ads/build.gradle]: Good call, switched this over to use the module-wide versioning

File: buildSrc/src/main/groovy/com.google.api-ads.java-conventions.gradle [hunk 2]
```diff
@@ -35,7 +35,7 @@ repositories {
 }
 
 group = 'com.google.api-ads'
-version = '14.0.1-SNAPSHOT'
+version = '15.0.0'
```
Comment (jradcliff) [inline buildSrc/src/main/groovy/com.google.api-ads.java-conventions.gradle:38]: Thanks for centralizing this. It should be `15.0.1-SNAPSHOT` to match the pom though, correct?

Comment (nwbirnie) [reply buildSrc/src/main/groovy/com.google.api-ads.java-conventions.gradle:38]: Well the release that just went live is 15.0.0, and this change was intended to be used to release the shadow jar. You are correct though that this should be 15.0.1-SNAPSHOT. I could complete the release and bump again?

Comment (jradcliff) [reply buildSrc/src/main/groovy/com.google.api-ads.java-conventions.gradle:38]: That SGTM.

**Review by jradcliff — APPROVED** [review]

**Review by devchas — APPROVED** [review]

**Pull request merged**